retrofit-adapters/rxjava/src/test/java/retrofit/ObservableCallAdapterFactoryTest.java

```diff
@@ -0,0 +1,183 @@
+package retrofit;
+
+import com.google.gson.reflect.TypeToken;
+import com.squareup.okhttp.MediaType;
+import com.squareup.okhttp.ResponseBody;
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.List;
+import org.junit.Test;
+import retrofit.converter.GsonConverter;
+import rx.Observable;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+public final class ObservableCallAdapterFactoryTest {
+  private final ObservableCallAdapterFactory factory = ObservableCallAdapterFactory.create();
+
+  @Test public void bodySuccess200() {
+    Type type = new TypeToken<Observable<String>>() {}.getType();
+    CallAdapter adapter = factory.get(type);
+    Observable<String> observable = (Observable<String>) adapter.adapt(new EmptyCall() {
+      @Override public void enqueue(Callback<Object> callback) {
+        callback.success(Response.fromBody("Hi"));
+      }
+    });
+    String body = observable.toBlocking().first();
+    assertThat(body).isEqualTo("Hi");
+  }
+
+  @Test public void bodySuccess404() {
+    Type type = new TypeToken<Observable<String>>() {}.getType();
+    CallAdapter adapter = factory.get(type);
+    Observable<String> observable = (Observable<String>) adapter.adapt(new EmptyCall() {
+      @Override public void enqueue(Callback<Object> callback) {
+        callback.success(Response.fromError(404, ResponseBody.create(MediaType.parse("application/json"), "Hi"), new GsonConverter()));
```

Comment:
Strange to have a non-JSON body here. Shouldn't it be text/plain?

(I'm assuming the converter is only exercised on 2xx responses)